scripts/zones/Chateau_dOraguille/npcs/_6h0.lua

```diff
@@ -41,7 +39,7 @@ function onTrigger(player,npc)
         player:startEvent(547);
     elseif (currentMission == tpz.mission.id.sandoria.RANPERRE_S_FINAL_REST and MissionStatus == 0) then
         player:startEvent(81);
-    elseif (currentMission == tpz.mission.id.sandoria.RANPERRE_S_FINAL_REST and MissionStatus == 4 and Wait1DayRanperre ~= osdate) then -- Ready now.
+    elseif (currentMission == tpz.mission.id.sandoria.RANPERRE_S_FINAL_REST and MissionStatus == 4 and player:needToZone() == false) then -- Ready now.
         player:startEvent(21);
```

Comment:
I don't think either of these lines are required anymore, once the `MissionStatus` var is correctly being set by the guard.